docs/software/system_stats.md

```diff
@@ -197,8 +197,12 @@ Netdata is a slick and feature-rich system stats monitor, with web interface.
     `/etc/netdata/netdata.conf`  
     and change the line  
     `bind socket to IP = 127.0.0.1`  
-    to match either the local network IP or static public IP of your server depending on your needs.  
-    Alternatively comment it e.g. if your server does not have a static public IP but you require remote access.
+    to match either the local network IP or static public IP of your dietpi server depending on your needs.
+    Alternatively comment it e.g. if your server does not have a static public IP but you require remote access. (Probably not a good idea. See "Security hardening" in next tab.)
+Once saved, you will need to restart the service to implement the changes by entering the below into the terminal:
+```bash
```

Comment:
Just to satisfy lint checks, please add an additional empty line before the code fence and use `sh` as language where bash is not strictly required.
````

```sh
````

Reply:
OK thank you for the feedback. :) I have [igorshubovych/markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli) installed but I don't use it. I will try next time unless there is a different one that's prefered.

Reply:
Since the repository has a GitHub action included, you should be able to enable GitHub actions for your fork to have all checks done automatically. But not required now, just as general info.